### PR TITLE
[ricos-converters] feat: add missing node converters

### PR DESCRIPTION
**Pull request by shlomo-gangul**

### CODE REVIEW

File: packages/ricos-converters/web/src/nodes/blockquote-converter.spec.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import type { BlockquoteNode } from 'ricos-content';
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/blockquote-converter.spec.ts:1]: Please revert all change in this file, unneeded

File: packages/ricos-converters/web/src/nodes/bulleted-list-converter.spec.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { bulletedListConverter } from './bulleted-list-converter';
+
+describe('bulletedListNode converter', () => {
+  const tiptapNode = {
+    type: Node_Type.BULLETED_LIST,
+    attrs: {
+      indentation: 2,
+      id: '1',
+    },
+    content: [] as [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/bulleted-list-converter.spec.ts:13]: Please check schema of bulleted list, it contains content/nodes

File: packages/ricos-converters/web/src/nodes/code-block-converter.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+export const codeBlockConverter: TiptapNodeConverter = {
+  toTiptap: {
+    type: Node_Type.CODE_BLOCK,
+    convert: (node: CodeBlockNode) => ({
+      type: Node_Type.CODE_BLOCK,
+      attrs: {
+        ...node.codeBlockData,
+        id: node.id,
+      },
+      content: [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/code-block-converter.ts:15]: Again, code-block node contains nodes , similar to bulleted-list

**Review by maimonav — CHANGES_REQUESTED** [review]
Please follow my comments, you need to go over all node converters and double check if they contain content/nodes, if so you need to convert them as well, using visit method, check paragraph-converter [visit](https://github.com/wix/ricos/blob/master/packages/ricos-converters/web/src/nodes/paragraph-converter.ts#L21) method usage for example.

**Comment by oded-soffrin** [comment]
I was trying to see why the e2e fails, but it seem that it has to do with converting

Tracking back the failing test ("allow to apply indent on multiple text blocks") on the example app (https://rich-content-nodeconverters.surge.sh/#!!)
i've taken the test content (/e2e/tests/fixtures/text-blocks.json) and converted from draft to prose - and oops the list is gone
image

image

we can probably have unit test that check the convertion before finding it in e2e

File: packages/ricos-converters/web/src/nodes/bulleted-list-converter.spec.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import { bulletedListConverter } from './bulleted-list-converter';
+
+describe('bulletedListNode converter', () => {
+  const tiptapNode = {
+    type: Node_Type.BULLETED_LIST,
+    attrs: {
+      indentation: 2,
+      id: '1',
+    },
+    content: [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/bulleted-list-converter.spec.ts:13]: Test nested nodes

File: packages/ricos-converters/web/src/nodes/code-block-converter.spec.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      },
+      id: '20',
+    },
+    content: [],
+  };
+
+  const codeBlockNode: CodeBlockNode = {
+    type: Node_Type.CODE_BLOCK,
+    id: '20',
+    nodes: [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/code-block-converter.spec.ts:21]: Test text nodes

File: packages/ricos-converters/web/src/nodes/collapsible-list-converters.spec.ts [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      direction: CollapsibleListData_Direction.LTR,
+      containerData: {
+        alignment: TextStyle_TextAlignment.CENTER,
+        textWrap: true,
+      },
+      expandOnlyOne: false,
+      id: '95',
+    },
+
+    content: [] as [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/collapsible-list-converters.spec.ts:26]: Test nested nodes

File: packages/ricos-converters/web/src/nodes/list-converter.spec.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+import { ricosNodeVisitor, tiptapNodeVisitor } from '../tiptap-converters';
+import { listItemConverter } from './list-converters';
+
+describe('ListItem converter', () => {
+  const tiptapNode = {
+    type: Node_Type.LIST_ITEM,
+    attrs: {
+      id: '82',
+    },
+    content: [] as ParagraphNode[],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/list-converter.spec.ts:12]: Test nested nodes

File: packages/ricos-converters/web/src/nodes/ordered-list-converter.spec.ts [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+import { orderedListConverter } from './ordered-list-converter';
+
+describe('OrderedList converter', () => {
+  const tiptapNode = {
+    type: Node_Type.ORDERED_LIST,
+    attrs: {
+      indentation: 2,
+      id: '45',
+    },
+    content: [] as ListItemNode[],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/ordered-list-converter.spec.ts:13]: Test nested nodes

File: packages/ricos-converters/web/src/nodes/ordered-list-converter.ts [hunk 9]
```diff
@@ -0,0 +1,33 @@
+import type { OrderedListNode } from 'ricos-content';
+import type { OrderedListData } from 'ricos-schema';
+import { Node_Type } from 'ricos-schema';
+import type { TiptapNodeConverter, TiptapNode } from '../types';
+
+export const orderedListConverter: TiptapNodeConverter = {
+  toTiptap: {
+    type: Node_Type.ORDERED_LIST,
+    convert: (node: OrderedListNode) => ({
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/ordered-list-converter.ts:9]: add visit usage

File: packages/ricos-converters/web/src/nodes/ordered-list-converter.ts [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+        ...node.orderedListData,
+        id: node.id,
+      },
+      content: [] as [],
+    }),
+  },
+
+  fromTiptap: {
+    type: Node_Type.ORDERED_LIST,
+    convert: (node: TiptapNode) => {
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/ordered-list-converter.ts:21]: also here

File: packages/ricos-converters/web/src/nodes/paragraph-converter.spec.ts [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+  const tiptapNode = {
+    type: Node_Type.PARAGRAPH,
+    attrs: {
+      textStyle: {
+        textAlignment: TextStyle_TextAlignment.AUTO,
+      },
+      indentation: 0,
+      id: '1',
+    },
+    content: [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/paragraph-converter.spec.ts:16]: Test nested nodes

File: packages/ricos-converters/web/src/nodes/table-converter.spec.ts [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+          {
+            type: Node_Type.PARAGRAPH,
+            attrs: {
+              textStyle: {
+                textAlignment: TextStyle_TextAlignment.AUTO,
+              },
+              indentation: 0,
+              id: '109',
+            },
+            content: [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/table-converter.spec.ts:33]: Test nested nodes

File: packages/ricos-converters/web/src/nodes/table-converter.spec.ts [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+          {
+            type: Node_Type.PARAGRAPH,
+            attrs: {
+              textStyle: {
+                textAlignment: TextStyle_TextAlignment.AUTO,
+              },
+              indentation: 0,
+              id: '111',
+            },
+            content: [],
```
Comment (maimonav) [inline packages/ricos-converters/web/src/nodes/table-converter.spec.ts:61]: Test nested nodes

**Review by maimonav — CHANGES_REQUESTED** [review]
Follow my comments :)

**Comment by shlomo-gangul** [comment]
> 

done

**Review by maimonav — CHANGES_REQUESTED** [review]
Looks great 🔥 
Please update changelog.

File: CHANGELOG.md [hunk 14]
```diff
@@ -17,6 +17,11 @@
     Click to see more.
   </summary>
 
+## :house: added missing converters
+
+- `ricos-converters`
+  - [#4065](https://github.com/wix/ricos/pull/4065) add missing node converters
+
```
Comment (maimonav) [inline CHANGELOG.md:24]: It's `## :house: Internal`, and there is already a section in here, take a look below.

**Review by maimonav — APPROVED** [review]

**Pull request merged**